CASMCMS-9141: Update multitenancy docs to use BOS CLI shortcuts (#5383)
Cray-HPE/docs-csm: operations/multi-tenancy/Modify_a_Tenant.md

1. (`ncn-mw#`) Clear the desired state.

>     - For a single component changing tenants, then use the following command:

Author: ```suggestion
    - For a single component changing tenants, use the following command:
```

>     - If only a single component is changing tenants, then use the following command:

Reviewer: ```suggestion
    - For a single component changing tenants, then use the following command:
```

        ```bash
        cray bos v2 components update --enabled true --clear-pending-state <xname> 
        ```

>     - For multiple components changing tenants, then use the following command:

Author: ```suggestion
    - For multiple components changing tenants, use the following command:
```

>     - If multiple components are changing tenants, then use the following command:

Reviewer: ```suggestion
    - For multiple components changing tenants, then use the following command:
```

        ```bash
        cray bos v2 components updatemany --enabled true --clear-pending-state --filter-ids <xname1>,<xname2>,...
        ```

1. (`ncn-mw#`) Wait until the components reach a `stable` state.

    Because the previous step cleared the desired state, the `stable` state indicates that the components are powered off.

>     - To check a single component, use the following command:

Reviewer: ```suggestion
    - For checking a single component, use the following command:
```

        ```bash
        cray bos v2 components describe <xname> --format json | jq .status.status
        ```

>     - To check multiple components, use the following command:

Reviewer: ```suggestion
    - For checking multiple components, use the following command:
```
